### DISCUSSION TITLE
Idea de proyecto propio.

### DISCUSSION THREAD

**Discussion by AndrjsM** [Launch-X-Latam/MisionFrontEnd#163] [Q&A]
Hola, buenas noches.

Les quiero compartir mi practica hasta la fecha: https://github.com/AndrjsM/Market-Place/blob/main/MiniMarket.md

La idea es distinta a la solicitada, es una idea propia.

Agradezco cualquier comentario y feedback que deseen dar. 

Conforme avance ire haciendo los updates.

Saludos.

**Comment by doguedogue** [comment]
genial 🚀🚀🚀

**Reply by AndrjsM** [reply]
Gracias @doguedogue

**Comment by AlbertoSF99** [comment]
Buenas, te quedó excelente el diagrama de flujo del punto 2, ¿podrías decirme en que app/sitio web lo hiciste? Saludos.

**Reply by AndrjsM** [reply]
Gracias bro, utilicé https://lucid.app/ para el flujo
